serializers/yelpToLocationSerializer.js

```diff
@@ -0,0 +1,62 @@
+class YelpToLocationSerializer {
```

Comment:
Great add with the serializer. It's going to help us immensely with rendering searches on the FE